lib/generators/manageiq/provider/templates/app/models/%plugin_path%/inventory/persister.rb

```diff
@@ -1,19 +1,3 @@
 class <%= class_name %>::Inventory::Persister < ManageIQ::Providers::Inventory::Persister
   require_nested :<%= manager_type %>
-
-  def strategy
-    nil
-  end
-
-  def parent
-    manager.presence
-  end
-
-  def shared_options
-    {
-      :strategy => strategy,
-      :targeted => targeted?,
-      :parent   => parent
-    }
-  end
```

Comment:
Also this, [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister/builder/persister_helper.rb#L101-L109)